README.md

```diff
@@ -3,5 +3,12 @@
 [![Dance](https://github.com/FerretDB/dance/actions/workflows/dance.yml/badge.svg)](https://github.com/FerretDB/dance/actions/workflows/dance.yml)
 
 FerretDB integration testing tool named after [Ferret war dance](https://en.wikipedia.org/wiki/Weasel_war_dance).
+It runs integration tests of various software that uses MongoDB (such as [MongoDB Go driver](https://github.com/mongodb/mongo-go-driver), [Mongoose ODM](https://mongoosejs.com), etc) against both MongoDB and FerretDB, and compares results with ones expected by tests configurations.
+It is expected that most or all tests pass when run against MongoDB, so we mark a few or none tests as expected failures or skips in configuration.
+More tests fail (and are marked as expected failures in tests configuration) when run against FerretDB, but their number goes down over time.
 
-Work-in-progress.
+Dance also includes one additional set of integration tests (called `ferret`) that is written by FerretDB developers.
```

Comment:
I can see how a test set name `ferret` can be confused with a database name / set's configuration `ferretdb`, but I did not like any alternative names I could come up with: `our`, `own`, `self`, etc. Any ideas?

(For example, here is MongoDB Go driver's configuration for FerretDB: https://github.com/FerretDB/dance/blob/d72807d789121b160ddcda91b75d1254b345af1c/tests/mongo-go-driver.yml#L15)

Reply:
origin?

Reply:
homebrew?

Reply:
homebrew +1